examples/torch/dqn_atari.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+"""An example to train a task with DQN algorithm.
+
+Here it creates a gym environment CartPole, and trains a DQN with 50k steps.
+"""
+import click
+import gym
+import numpy as np
+import psutil
+import torch
+
+from garage import wrap_experiment
+from garage.envs import GymEnv
+from garage.envs.wrappers.clip_reward import ClipReward
+from garage.envs.wrappers.episodic_life import EpisodicLife
+from garage.envs.wrappers.fire_reset import FireReset
+from garage.envs.wrappers.grayscale import Grayscale
+from garage.envs.wrappers.max_and_skip import MaxAndSkip
+from garage.envs.wrappers.noop import Noop
+from garage.envs.wrappers.resize import Resize
+from garage.envs.wrappers.stack_frames import StackFrames
+from garage.experiment.deterministic import set_seed
+from garage.np.exploration_policies import EpsilonGreedyPolicy
+from garage.replay_buffer import PathBuffer
+from garage.sampler import FragmentWorker, LocalSampler
+from garage.torch import set_gpu_mode
+from garage.torch.algos import DQN
+from garage.torch.policies import DiscreteQFArgmaxPolicy
+from garage.torch.q_functions import DiscreteCNNQFunction
+from garage.trainer import Trainer
+
+hyperparams = dict(n_epochs=500,
```

Comment:
can you just make these the default parameters to your experiment function?

Reply:
Please see my other comment, though I'm happy to do this if you prefer.